app/public/css/0-1366px.css

```diff
@@ -1,28 +1,26 @@
-@media screen and (max-height: 1366px) {
-  .welcome {
-    position: relative;
-    margin-bottom: -112px;
-    margin-left: -56px;
-    margin-right: -56px;
-  }
+ .welcome {
```

Comment:
I would rather look for the root cause of the missing footer for big resolutions and fix the base CSS that remove the restriction on screen resolutions from a CSS meant for a certain resolution range (0-1366px) as its name suggests

Reply:
ok, I can update the base welcome-footer class with the correct styles for the footer and leave the rest within the original resolution range.